controllers/provisioning_controller.go

```diff
@@ -127,6 +123,7 @@ func (r *ProvisioningReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 	if baremetalConfig == nil {
 		// Provisioning configuration not available at this time.
 		// Cannot proceed wtih metal3 deployment.
+		r.Log.V(1).Info("Provisioning CR not found")
```

Comment:
We could end up here if the Provisioning CR is not found or if it ill-formed and cannot be read. The log msg here only mentions that the provisioning CR is not found.